app/mailers/spree/payment_mailer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Spree
+  class PaymentMailer < BaseMailer
```

Comment:
I think we can put this outside of the Spree namespace as well.

Reply:
c71e5037129c43f2431c4ab4037af7cea20f6e88